src/os/bluestore/BitAllocator.cc

```diff
@@ -0,0 +1,1256 @@
+/*
```

Comment:
please add the emacs/vi modelines at the top (top 2 lines of every other .cc or .h file in the repo).  they set tab stops at 2 chars to match the rest of the code base.